### ISSUE TITLE
[Adaptive Cards-Schema Explorer]: Schema Explorer Link tab is not highlighted even when it got selected from the Header Region.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7890]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Repro Steps:
1.Open above URL with V- credential.
2.Navigate to header controls using Tab key and activate 'Schema Explorer' link using Enter key.
3. Observe the issue in scan mode.

Actual Result:
"Schema Explorer" Link is not highlighted from the Header tabs.

Expected Result:
"Schema Explorer" Link tab should be highlighted with different colour or it should be Bold or it should have underlined as it is the selected tab from the header region.

User Impact:
Cognitive keyboard users will be impacted if the opened page link is not highlighted diffrently and they need to cross verify with the heading present in the main content page to know which link they have activated.

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="960" alt="Schema Explorer link is not getting highlighted as it got selected from the header" src="https://user-images.githubusercontent.com/104482612/190593148-5625503e-96a8-4b14-8fb2-00128454eddb.png">

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA
